AnnService/src/Core/BKT/BKTIndex.cpp

```diff
@@ -328,7 +321,7 @@ namespace SPTAG
             bool NeverDup(COMMON::QueryResultSet<T>& query, SizeType node, float score)
             {
                 query.AddPoint(node, score);
-                return false;
+                return true;
```

Comment:
This will mean it will always break on the first iteration, I don't think it's correct